contrib/scrooge/src/python/pants/contrib/scrooge/tasks/scrooge_gen.py

```diff
@@ -78,6 +78,13 @@ def register_options(cls, register):
             type=dict,
             help="Registered target types.",
         )
+        register(
+            "--allowlist-compiler-args",
+            advanced=True,
+            type=list,
+            default=["--java-passthrough"],
+            help="Don't report targets if these compiler args are missing.",
```

Comment:
It took me a bit to understand this help string. In my mind, `report` means "write metrics somewhere".
Could we expand this description a bit? Something like:
```suggestion
            help="Don't fail if these options are different between targets.\n"
                      "Usually, Scrooge requires all targets in the dependency tree to have the same options."
                      "However, discrepancies in options specified in this list will not cause the compiler to fail.",
```